test/libsolidity/syntaxTests/nameAndTypeResolution/589_large_storage_structs.sol

```diff
@@ -0,0 +1,18 @@
+contract C {
```

Comment:
Ideally the test should be grouped together with `506_large_storage_array_fine.sol`, ..., `510_large_storage_array_mapping.sol`. Would test re-numbering make sense?

Reply:
The ``nameAndTypeResolution`` test directory was auto-generated from a ``.cpp`` file at some point - the numbers have no real relevance. So it's always better *not* to add new tests there, but rather add new tests in a more structured part of ``syntaxTests/``, while potentially moving related tests from ``nameAndTypeResolution`` as well.

None of it is really important or should be blocking this, though.